data/decks/default.lua

```diff
@@ -0,0 +1,40 @@
+mesh = "../meshes/beam-hex.mesh"
+
+-- serial and parallel refinement levels
+ser_ref_levels = 0
+par_ref_levels = 0
+
+-- polynomial interpolation order
+order = 1
```

Comment:
This is related with `nonlinear_solid`. We could have multiple physics modules with different polynomial orders.